test/unit/light/unit_ParlioLedDriver.cpp

```
@@ -240,3 +240,20 @@ TEST_CASE("ParlioLedDriver loopbackRxPin tracks the loopbackTest toggle") {
     }
     CHECK(found);
 }
+
+// loopbackTxPin (optional lane-0 TX override) is bound always, hidden until the
+// test is on — same conditional-control contract as loopbackRxPin. The override's
+// lane-0 substitution is hardware-only (parlioLanes==0 on desktop); the visibility
+// contract is host-testable here.
+TEST_CASE("ParlioLedDriver loopbackTxPin tracks the loopbackTest toggle") {
+    mm::ParlioLedDriver d;
```

Clang (14.0.6)

[warning] variable name 'd' is too short, expected at least 3 characters

(readability-identifier-length)